Framework/API/src/MatrixWorkspace.cpp

```diff
@@ -2047,6 +2055,80 @@ IPropertyManager::getValue<Mantid::API::MatrixWorkspace_const_sptr>(
   }
 }
 
+template <>
```

Comment:
Do we have to add bits of code like the one below for all workspace types (I saw it above also for `IEventWorkspace`)? Can that be avoided?